src/utilities/create-compilers.js

```diff
@@ -43,9 +44,11 @@ module.exports = ({ cwd, env }) => {
 
   const serverCompiler = createCompiler(configServerDefault)
   const clientCompiler = createCompiler(configClientDefault)
+  const devCompiler = createCompiler(configDevDefault)
```

Comment:
The dev config and compiler is very different from the other two, so also warrants it's own file, but does use some of the shared module config.